Xamarin.Forms.Core/FontElement.cs

```diff
@@ -1,8 +1,10 @@
+using System.ComponentModel;
 using Xamarin.Forms.Internals;
 
 namespace Xamarin.Forms
 {
-	static class FontElement
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public static class FontElement
```

Comment:
@VincentDondain Like anything that inherits from `IFontElement` uses those bindable properties https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core/Label.cs#L38, technically you could just change all the places that it's using FontElement.<Property> to Label.<Property> (for example).

Reply:
@jsuarezruiz I think you mean me, @VincentH-Net ?

Reply:
Yes, updated the comment!.

Reply:
Thx @jsuarezruiz I built against the packages from this PR; results:
IPaddingElement` also needs to be made public - I missed that one when documenting, my bad.

Better keep those changes; reverting them would require duplication of all extension methods targeting `IPaddingElement` and `IFontElement` for all view that implement them. Even worse, it would make those methods unavailable for any (current/future) views in XCT or other libraries that have padding and font properties

Reply:
@jsuarezruiz could you make `IFontElement` public? Thx

Reply:
I would wait until we can have default interface members in XF, and get rid of the static *Element types

Reply:
@StephaneDelcroix , @jsuarezruiz The decision whether to wait/release should take @davidortinau 's goal into account,  not to delay XF5 stable longer than needed:
> What's the status on moving all the planned things from xamarin/xamarin.forms to xamarin/xamarincommunitytoolkit?
@VincentH.NET @jfversluis @ShaneNeuville @samhouts @redth (tagging anyone that might know so we don't delay XF 5 stable longer than needed)
(Quote from [David's recent discus message](https://discord.com/channels/732297728826277939/738065609740517467/770032960429621268))

Please let me know when / what you decide; I am mostly done with the move and could complete quickly once this PR incl public `IPaddingElement` is merged

Reply:
@VincentH-Net for now can we just replace your FontElement.BP and PaddingElement.BP with 

Label.BP

I know it's kind of ugly and it's not ideal but it's functionally the same and later down the road we can do what @StephaneDelcroix suggests

Reply:
Yes, the @PureWeen suggestion is what I tried to agree with my first question. We can maintain the static Element types internal and open all the necessary interfaces. From the Markup extensions, replace `FontElement.{Property}` by `Label.{Property}`. @VincentH-Net What do you think?

Reply:
Yes that is fine by me. I don't mind some internal ugliness, as long as it does not hurt the XCT users.

So OK to keep the static *Element types internal and the interfaces public.
(apologies that I am a bit late in responding; I was up to my eyebrows in the PR)